config/locales/devise.en.yml

```diff
@@ -92,9 +92,10 @@ en:
         page_heading: Sign in
         password: Password
         sign_in: Sign in
-      signed_in: ""
-      signed_out: ""
-      already_signed_out: ""
+      user:
+        signed_in: Signed in successfully
```

Comment:
Do you have any idea why these messages were blanked? Does this affect behaviour in any way.

For reference see #647, although that doesn't really say much.